File: webdav/perl/create/webdav_create.pl
#! /usr/bin/perl
#
#	webdav_create.pl
#
#					Jan/24/2012
# -------------------------------------------------------
use	strict;
use	warnings;
use	utf8;
use	Encode;
use	JSON;
#
use lib '/var/www/data_base/common/perl_common';
use text_manipulate;
use uri_get;
#
# -------------------------------------------------------
print (encode ('utf-8',"*** 開始 ***\n"));
#
#
my $uri = 'http://host_dbase:3004/city/tokyo.json';
#
my %dict_data = data_prepare_proc ();
#
my $json_str = encode_json (\%dict_data);
#
uri_get::uri_put_proc ($uri,$json_str);
#
print (encode ('utf-8',"*** 終了 ***\n"));
#
# -------------------------------------------------------
sub data_prepare_proc
{
	my %dict_aa;

	%dict_aa = text_manipulate::dict_append_proc
		('t1351','八王子',84259,'2005-9-12',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1352','立川',14278,'2005-3-15',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1353','武蔵野',58641,'2005-6-17',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1354','三鷹',46432,'2005-8-22',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1355','青梅',27349,'2005-2-28',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1356','府中',36148,'2005-5-9',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1357','昭島',35792,'2005-4-10',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1358','調布',85146,'2005-10-8',%dict_aa);
	%dict_aa = text_manipulate::dict_append_proc
		('t1359','町田',64827,'2005-5-21',%dict_aa);

	return	%dict_aa;
}

# -------------------------------------------------------

File: webdav/node/create/webdav_create.js
#! /usr/bin/nodejs
// ---------------------------------------------------------------
//	webdav_create.js
//
//					Jul/20/2012
//
// ---------------------------------------------------------------
var text_manipulate = require('/var/www/data_base/common/node_common/text_manipulate');
var http_manipulate = require('/var/www/data_base/common/node_common/http_manipulate');
// ---------------------------------------------------------------
console.log ("*** 開始 ***");
//

var dict_aa = data_prepare_proc ();
var json_str = JSON.stringify(dict_aa);

http_manipulate.put_proc ('host_dbase',3004,'/city/tokyo.json',json_str);

console.log ("*** 終了 ***");


// ---------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ();

	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1351','八王子',75214,'1950-9-12');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1352','立川',31587,'1950-3-15');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1353','武蔵野',65278,'1950-10-2');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1354','三鷹',38695,'1950-6-22');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1355','青梅',42158,'1950-8-14');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1356','府中',67892,'1950-9-12');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1357','昭島',38751,'1950-3-21');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1358','調布',52486,'1950-7-26');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1359','町田',26547,'1950-10-2');


	return	dict_aa;
}	

// ---------------------------------------------------------------
